### PR TITLE
new YAML TJB

### PR DESCRIPTION
**Pull request by tjbarnum13**
create thiazole.yml, thiophene.yml

### CODE REVIEW

File: data/thiazole.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# thiazole.yml
+A: 8529.447
```
Comment (bchangala) [inline data/thiazole.yml]: 8529.447921 (any reason for truncating last 3 digits?)

File: data/thiazole.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+chi_bb-chi_cc: -2.5780
+u_a: 1.286
+u_b: 0.966
+rep: Ir
+doi: [10.1063/5.0057221, 10.1016/0022-2852(62)90230-8, 10.1515/zna-1993-1211]
+name: thiazole
+formula: C3H3NS
+smiles: C1=CSC=N1
+notes: Constants from Ref 1. Dipole from Ref 2. N hfs from Ref 3.
+author: TJB
```
Comment (bchangala) [inline data/thiazole.yml:29]: are quotes required for non-numerical entries? (rep, doi, name, formula, smiles, notes, author...)

File: data/thiophene.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+phi_JK: -0.0000859E-6
+phi_K: 0.003184E-6
+u_a: 0.55
+rep: Ir
+doi: [10.1063/5.0055267, 10.1016/0022-2852(72)90141-5]
+name: thiophene
+formula: C4H4S
+smiles: C1=CSC=C1
+notes: Constants from Ref 1. Dipole from Ref 2.
+author: TJB
```
Comment (bchangala) [inline data/thiophene.yml:24]: Same question re. quotes.

**Review by bchangala — CHANGES_REQUESTED** [review]
See in-line comments.

File: data/thiophene.yml (hunk 3 above)
Comment (laserkelvin) [reply data/thiophene.yml:24]: No quotes are fine - it still gets parsed as a string

File: data/thiazole.yml (hunk 2 above)
Comment (tjbarnum13) [reply data/thiazole.yml:29]: no, quotes are not required.

Comment (laserkelvin) [reply data/thiazole.yml:29]: Nope, as long as it gets parsed as whole strings it's okay, and they do

**Pull request merged**